aries-site/src/examples/components/namevaluelist/Examples/NameValueListMultipleDefaultExample.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Box, Heading, NameValueList, NameValuePair, Tab, Tabs } from 'grommet';
+import { kubernetesData, serverData, tagsData } from '../data';
+
+export const NameValueListMultipleDefaultExample = () => {
+  const [index, setIndex] = React.useState();
+  const onActive = nextIndex => setIndex(nextIndex);
+  return (
+    <>
+      <Heading level={1} size="small">
+        Demo_Cluster_5
+      </Heading>
+      <Tabs activeIndex={index} onActive={onActive} justify="start">
+        <Tab title="Overview">
+          <Heading level={2} size="small">
+            Details
+          </Heading>
+          <NameValueList>
```

Comment:
I believe the gaps on these should be "small".

```suggestion
          <NameValueList gap="small">
```

Also, does this make sense to default to small in the theme?

Reply:
NVM, resolved on call.